RZViewActions/UIView+RZViewActions.m

```diff
@@ -128,6 +142,17 @@ - (void)_runWithCompletion:(RZViewActionCompletion)completion
 
 @end
 
+#pragma mark - RZViewSpringAction
+
+@implementation RZViewSpringAction
+
+- (void)_runWithCompletion:(RZViewActionCompletion)completion
```

Comment:
Yes, @jvisenti, it's called out in the ["Naming Methods" section of Coding Guidelines for Cocoa](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html)

    Don’t use the underscore character as a prefix for your private methods. Apple reserves this convention.